### ISSUE TITLE
Sentry error integration

### ISSUE THREAD

**Issue by paraschivionut** [cami-project/cami-project#76]
## Why

In apps in general, errors can exist and they usually throw large stack exceptions. These are usually thrown to the standard output or redirected to a file on the server, which can be very hard to follow if we want to track some bugs/errors.

## What

Integrate the error integration library and configure it in the next projects:
- [x] frontend
- [x] medical_compliance

## Notes

For this task we can use [Sentry](https://sentry.io/for/django/). A small research for other options would be also good in this case.

**Comment by cretualexandru** [comment]
Issue fixed in PR #90.

**Comment by aismail** [comment]
@cretualexandru just use hub's issue to PR conversion from now on. Thanks!
